test/extended/builds/run_fs_verification.go

```diff
@@ -144,6 +144,7 @@ valid_fields.json
 
 		g.Describe("do not have unexpected content", func() {
 			g.It("using a simple Docker Strategy Build", func() {
+				g.Skip("Bug 1959564: skipping while additional content in /run is investigated")
```

Comment:
How about using `result.Flakef` so that we get a clear signal?

Reply:
This isn't a flaky test - cri-o broke a conformance test for builds. This unblocks teams from merging fixes.

It will still appear in the list of skipped tests - I do concede that the skipped list can be quite large.